identity/src/rsa.rs

```diff
@@ -326,9 +350,9 @@ mod tests {
     use super::*;
     use quickcheck::*;
 
-    const KEY1: &[u8] = include_bytes!("test/rsa-2048.pk8");
-    const KEY2: &[u8] = include_bytes!("test/rsa-3072.pk8");
-    const KEY3: &[u8] = include_bytes!("test/rsa-4096.pk8");
+    const KEY1: &[u8] = include_bytes!("test/private_key_rsa-2048.pk8");
+    const KEY2: &[u8] = include_bytes!("test/private_key_rsa-3072.pk8");
+    const KEY3: &[u8] = include_bytes!("test/private_key_rsa-4096.pk8");
```

Comment:
Is this rename necessary? I'd prefer the diff to be minimal.

Reply:
Then let me undo it.